.zenodo.json

```diff
@@ -13,15 +13,15 @@
     "contributors": [
         {
             "name": "Roger Blench",
-            "type": "Other"
+            "type": "Editor"
```

Comment:
Please consult with @LinguList about the role of Roger Blench for this repo

Reply:
@LinguList is `Editor` ok for Roger Blench or should I add another tag?

Reply:
Other, please